.rubocop.yml

```diff
@@ -1,6 +1,6 @@
 # TODO: remove this file by either moving cops here or fixing code
 inherit_from:
-  - .rubocop_todo.yml
+  - https://raw.githubusercontent.com/theforeman/foreman/3.8-stable/.rubocop.yml
```

Comment:
This is really cool, I didn't know you could pull in another repos `.rubocop.yml` I wonder if we should do this with `hammer-cli-katello` when we get time